README.md

```diff
@@ -69,6 +69,7 @@ COMMANDS:
      addons-add        Provision an add-on for your application
      addons-remove     Remove an existing addon from your app
      addons-upgrade    Upgrade or downgrade an add-on attached to your app
+     addons-info       Display infos about add-ons attached to your app
```

Comment:
The word "information" is uncountable in English. Moreover this call is about a single addon, hence I would remove the `s`
```suggestion
     addons-info       Display info about add-on attached to your app
```

Reply:
I thought so too. But since all the others commands (addons-add, addons-remove, addons-upgrade...) have an `s` I got confused.